job-form.html

```diff
@@ -135,6 +135,15 @@ <h1>Submit a Job</h1>
 			name="fields[how_to_apply]">
 		  <small>Markdown is okay</small>
 		</div>
+		<div class="form-group">
+		      <p class="help-block"></p>
+		      <input class="form-control"
+		      id="how_to_apply"
```

Comment:
The id `how_to_apply` is duplicated from above. Could you change it to `github_handle`?

Reply:
```suggestion
		      id="github_handle"
```